test/structs_common/test_exception.py

```diff
@@ -75,7 +75,8 @@ def process_test():
 
                 cause = random.choice(list(ExceptionCause))
                 report_rob = random.randint(0, self.rob_max)
-                report_arg = {"cause": cause, "rob_id": report_rob}
+                report_pc = random.randint(0, 2**self.gp.isa.xlen - 1)
```

Comment:
My favourite nitpick :)
```suggestion
                report_pc = random.randrange(2**self.gp.isa.xlen)
```